nodes.erl

```diff
@@ -73,7 +78,7 @@ query_to_commit(OtherNodes) ->
     broadcast(OtherNodes, {query, self()}).
 
 broadcast(Nodes, Message) ->
-    lists:map(fun(Node) -> Node ! Message end, Nodes).
+    [Node ! Message || Node <- Nodes].
```

Comment:
For me it's much easier to use list comprehensions in such cases since it really dries up your code.

I prefer `lists:map/2` when the function to call on the lists items is complex and in such cases I bind the function to a variable.

WTG with elixir...it's the first step towards erlang :D